rero_ils/config.py

```diff
@@ -4396,3 +4396,6 @@ def search_type(field):
 RERO_FILES_RECORD_FILE_SERVICE_CONFIG = (
     "rero_ils.modules.files.services.RecordFileServiceConfig"
 )
+RERO_ILS_APP_FILES_UI_MAX = 600
+RERO_ILS_APP_FILES_MAX = RERO_ILS_APP_FILES_UI_MAX * 3 + 100
```

Comment:
If you want to make this variable value in function of the UI variable you need to put it into the max_files_count class method.